DRAFT_RELEASE_NOTES.md

```diff
@@ -14,6 +14,13 @@ There are no known issues with the 1.9.0 release.
 | Date<br/>identified | Vulnerability | Severity | Package | Affected <br/>versions | CVE | Fixed <br/>in |
 |---------------------|---------------|----------|---------|------------------------|-----|---------------|
 
+# Recommended Kubernetes Version
+aiSSEMBLE recommends any consumer be on a minimum Kubernetes version of 1.30. This is due to security 
+findings in 1.29. For more information on Kubernetes current security findings, view their 
+[CVE feed](https://kubernetes.io/docs/reference/issues-security/official-cve-feed/). If using EKS, please follow 
```

Comment:
`If using EKS`. Should this be `If using AWS EKS`?

Reply:
sure

Reply:
Oh I guess EKS is specific to AWS

Reply:
Yeah I think both work. Doesnt hurt to put AWS EKS